components/SubmissionsTable/index.tsx

```diff
@@ -22,20 +23,28 @@ export const SubmissionsTable = ({
 
   const searchableSubmissions = useMemo(
     () =>
-      submissions.filter((submission) => {
-        // hide any restricted records unless the user has permission to see them
-        if (
-          !user?.hasUnrestrictedPermissions &&
-          submission.residents.every((resident) => resident.restricted === 'Y')
-        )
-          return false;
+      submissions
+        // augment each one with its form
+        .map((submission) => ({
+          ...submission,
+          form: forms.find((form) => form.id === submission.formId),
+        }))
```

Comment:
Is this somehow pairing a submission with its original form?

Reply:
yes - the forms are json files stored in the git repo. the submission stores a `formId` which we can relate back to a particular form, particularly the human-readable name of the form